docs/getting-started/installation/install-redis-from-source.md

```diff
@@ -30,23 +42,44 @@ make
 
 If the compile succeeds, you'll find several Redis binaries in the `src` directory, including:
 
-* **redis-server**: the Redis Server itself
-* **redis-cli** is the command line interface utility to talk with Redis.
+* **redis-server**: The Redis Server itself
+* **redis-cli**: The command line interface utility to talk with Redis.
+* **redis-benchmark**: The command line tool for benchmarks
```

Comment:
let's also mention 
redis-sentinel
redis-check-rdb
redis-check-aof

Reply:
Need the descriptions...